src/renderer/equalizerApi.ts

```diff
@@ -178,153 +197,42 @@ export const getFrequency = (index: number): Promise<number> => {
  * @param {frequency} index - index of the slider being adjusted
  */
 export const setFrequency = (index: number, frequency: number) => {
-  const channel = `setFrequency${index}`;
-  window.electron.ipcRenderer.sendMessage('peace', [
-    channel,
-    100 + index,
-    6,
-    frequency,
-  ]);
-
-  const responseHandler = buildResponseHandler<number>((result, resolve) => {
-    resolve(peaceFrequencyOutputToNormal(result));
-  });
-  return promisifyResult(responseHandler, channel);
-};
-
-/**
- * Get program state for Peace. We will use this as a health check call
- * @returns { Promise<void> } exception if Peace is not okay.
- */
-export const getProgramState = (): Promise<void> => {
-  const channel = 'getProgramState';
-  window.electron.ipcRenderer.sendMessage('peace', [channel, 0, 0]);
-  return promisifyResult(setterResponseHandler, channel);
-};
-
-/**
- * Enable Equalizer
- * @returns { Promise<void> } exception if failed.
- */
-export const enableEqualizer = (): Promise<void> => {
```

Comment:
Moved `enableEqualizer`, `disableEqualizer` and `getEqualizerStatus` to higher up in the file to match the order in which the corresponding channels are defined in `ChannelEnum`